purchase_order_reorder_lines/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Purchase order lines with sequence number
+=========================================
+
+Provide a new field on the purchase order form, allowing to manage the order of the lines
+order.
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```

Comment:
Include a section Installation explaining that the module stock_picking_reorder_lines is needed and point to OCA/stock-logistics-workflow repo.